packages/target-electron/src/windows/video-call.ts

```diff
@@ -572,10 +632,72 @@ function isOriginGood(url: string) {
   }
   return (
     urlParsed.protocol === SCHEME_NAME + ':' &&
-    urlParsed.host === DUMMY_HOST_NAME &&
+    parseHost(urlParsed.host) != null &&
     urlParsed.port === ''
   )
 }
+function formatHost(
```

Comment:
```suggestion
/**
 * @returns the host string in format
 *  `{callMessageId|none}.{chatId|none}.{accountId}.calls-webapp-dummy-host`
 */
 function formatHost(
```

Reply:
I force-pushed, adding `type HostStr`. See if it's good.